rep/replay.go

```diff
@@ -139,7 +140,11 @@ func (r *Replay) Compute() {
 
 		switch r.Header.Type {
 		case repcore.GameTypeUMS:
-			r.computeUMSTeams()
+			if strings.Contains(strings.ToLower(r.MapData.Name), "[ai]") {
```

Comment:
the exact map name of the said [AI] team splitting team is: "\u0004[AI] \u0006BGH \u00053.0 BETA"

May want to use this exact name to differentiate, it works for 24H mmr bot on our side, because this would be the only map we would use, but if it is for general use, it may needs to be exact on the map name.

I've also tested the result looking good on the test on my side as well.

Reply:
It works for map names containing "[AI]".
What are other cases you are talking about?

Reply:
I have not checked, but I am worried about other map that has tag with [AI] but the alliance command and functionality might be totally different since it is use map setting ,it could be anything essentially, but we know for sure this map works.  Or the list of the [AI] map that you have work the same way which is the random team method.

But I suppose you checked with the alliance command in the code later, so if it is this type of the map then it will be recognized.  So yeah, if that's covered then there's no issue I'd raise here.